docs/docs/schema-customization.md

```diff
@@ -206,6 +211,7 @@ Note that you don't need to explicitly provide the Node interface fields (`id`,
 > If you wonder about the exclamation marks - those allow
 > [specifying nullability](https://graphql.org/learn/schema/#lists-and-non-null)
 > in GraphQL, i.e. if a field value is allowed to be `null` or not.
+In simple terms the exclamatory mark doesn't allow the value to be a null and always caters to make it remain as a non-nullable value.
```

Comment:
This is redundant with the above line.

Reply:
We also try to avoid terms like "simple", "easy", etc. since everyone has a different definition of what's easy. See the Gatsby Style Guide for more info: https://www.gatsbyjs.org/contributing/gatsby-style-guide/#avoid-easy-and-simple